packages/lodestar-cli/src/cmds/account/cmds/validator/voluntaryExit.ts

```diff
@@ -0,0 +1,122 @@
+
+  examples: [
+    {
+      command: "account validator voluntary-exit --publicKey 0xF00",
+      description: "Perform a voluntary exit for the validator who has a public key 0xF00",
+    },
+  ],
+
+  options: {
+    server: validatorOptions.server,
+    force: validatorOptions.force,
+
+    publicKey: {
+      description: "The public key of the validator to voluntarily exit",
+      type: "string",
+    },
+  },
+
+  handler: async (args) => {
+    await initBLS();
+
+    const force = args.force;
+    const server = args.server;
+    let publicKey = args.publicKey;
+    const logger = new WinstonLogger();
+    const accountPaths = getAccountPaths(args);
+    const validatorDirManager = new ValidatorDirManager(accountPaths);
+
+    if (!publicKey) {
+      const publicKeys = readdirSync(accountPaths.keystoresDir);
+
+      const validator = await inquirer.prompt([
+        {
+          name: "publicKey",
+          type: "list",
+          message: "Which validator do you want to voluntarily exit from the network?",
+          choices: [...publicKeys],
+        },
+      ]);
+      publicKey = validator.publicKey;
+    }
+
+    const confirmation = await inquirer.prompt([
+      {
+        name: "choice",
+        type: "list",
+        message: `Are you sure you want to permantently exit validator ${publicKey} from the ${args.network} network?  WARNING: THIS CANNOT BE UNDONE.`,
+        choices: ["NO", "YES"],
+      },
+    ]);
+
+    if (confirmation.choice === "NO") return;
+
+    logger.info(`Initiating voluntary exit for validator ${publicKey}`);
+
+    const secretKey = await validatorDirManager.decryptValidator(publicKey, {force});
+    if (!secretKey) throw new YargsError("No validator keystores found");
+    logger.info(`Decrypted keystore for validator ${publicKey}`);
+
+    const config = getBeaconConfigFromArgs(args);
+    const api = new ApiClientOverRest(config, server, logger);
+    await api.connect();
+
+    const validator = await api.beacon.state.getStateValidator("head", config.types.BLSPubkey.fromJson(publicKey));
+    if (!validator) throw new YargsError("No validator found in validator store.");
+
+    const epoch = api.clock.currentEpoch;
+
+    const voluntaryExit = {
+      // Minimum epoch for processing exit
+      epoch,
+      validatorIndex: validator.index,
+    };
+
+    const state: BeaconState = config.types.BeaconState.tree.defaultValue();
+    state.fork = (await api.beacon.state.getFork("head")) || state.fork;
+    state.genesisValidatorsRoot = (await api.beacon.getGenesis())?.genesisValidatorsRoot || state.genesisValidatorsRoot;
+
+    const domain = getDomain(config, state, DomainType.VOLUNTARY_EXIT, epoch);
+    const signingRoot = computeSigningRoot(config, config.types.VoluntaryExit, voluntaryExit, domain);
+
+    const signedVoluntaryExit: SignedVoluntaryExit = {
+      message: voluntaryExit,
+      signature: secretKey.sign(signingRoot).toBytes(),
+    };
```

Comment:
Can you move voluntary exit creation process to lodestar-validator as it would allow users of lodestar-validator to use that as well?

Something like:
```
const validator = new LodestarValidator(...)

await validator.voluntaryExit()
```